sql-plugin/src/main/spark330db/scala/org/apache/spark/sql/rapids/execution/GpuBroadcastHashJoinExec.scala

```diff
@@ -169,6 +171,7 @@ case class GpuBroadcastHashJoinExec(
     val rdd = streamedPlan.executeColumnar()
     val localBuildSchema = buildPlan.schema
     val localBuildOutput = buildPlan.output
+    val localIsNullAwareAntiJoin = isNullAwareAntiJoin
```

Comment:
Could we remove this local variable?

Reply:
These local variables can avoid the entire outer exec instance being serialized to executors.